addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -342,9 +337,10 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
 
         if (command != null) {
+            stopRefreshThread();
```

Comment:
Is that really the best solution? To cancel a scheduled job and reschedule it again, just to execute a command in between?
Shouldn't at least a running task be allowed to end gracefully first instead of being interrupted?